tests/Unit/BankDataConverterTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace WMDE\Fundraising\Tests\Unit;
+
+use WMDE\Fundraising\Frontend\BankDataConverter;
+use WMDE\Fundraising\Frontend\Domain\BankData;
+use WMDE\Fundraising\Frontend\FunFunFactory;
+use WMDE\Fundraising\Frontend\Tests\TestEnvironment;
+
+/**
+ * @covers WMDE\Fundraising\Frontend\BankDataConverter
+ *
+ * @licence GNU GPL v2+
+ * @author Christoph Fischer <christoph.fischer@wikimedia.de
+ */
+class BankDataConverterTest extends \PHPUnit_Framework_TestCase {
+
+	/** @var FunFunFactory */
+	private $factory;
+
+	public function setUp() {
+		if ( !function_exists( 'lut_init' ) ) {
+			$this->markTestSkipped( 'The konto_check needs to be installed!' );
+		}
+		$this->factory = TestEnvironment::newInstance()->getFactory();
```

Comment:
My idea for test organization is to not ever access the factory or environment from a unit test. After all, if you need it, it is supposedly not actually a unit test.